rotkehlchen/db/dbhandler.py

```diff
@@ -2334,6 +2350,35 @@ def get_exchange_credentials_extras(self, name: str, location: Location) -> Dict
 
         return extras
 
+    def set_binance_pairs(self, name: str, pairs: List[str], location: Location) -> None:
+        cursor = self.conn.cursor()
+        data = json.dumps(pairs)
+        cursor.execute(
+            'INSERT OR REPLACE INTO user_credentials_mappings '
+            '(credential_name, credential_location, setting_name, setting_value) '
+            'VALUES (?, ?, ?, ?)',
+            (
+                name,
+                location.serialize_for_db(),
+                BINANCE_MARKETS_KEY,
+                str(data),
```

Comment:
nitpick: Why do you need `str()` here. Json dumps should always return a string.